src/lib/model/ContentItem.ts

```diff
@@ -153,8 +158,11 @@ abstract class BaseContentItem extends HalResource {
      * You must provide the current version number in the mutation
      * to avoid overwriting other user's changes.
      */
-    update: (mutation: ContentItem): Promise<ContentItem> =>
-      this.updateResource(mutation, ContentItem),
+    update: (
+      mutation: ContentItem,
+      params?: { ignoreValidation?: boolean }
```

Comment:
Should this match the hub setting `ignoreSchemaValidation` or is it intentionally different?

Reply:
yeah, should be `ignoreSchemaValidation`